packages/core/src/checkbox/checkbox.stories.ts

```diff
@@ -37,8 +37,8 @@ export function checkbox() {
   return html`
     <cds-checkbox>
       <label>checkbox</label>
-      <input type="checkbox" checked />
-      <cds-control-message>message message</cds-control-message>
+      <input type="checkbox" />
```

Comment:
Are we no longer demo'ing how `checked` works from HTML?

Reply:
Typo I think will fix 👍

Reply:
fixed